freqtrade/strategy/informative_decorator.py

```diff
@@ -0,0 +1,129 @@
+from typing import Any, Callable, NamedTuple, Optional, Union
+
+from mypy_extensions import KwArg
```

Comment:
this line concerns me a bit ...

it's assuming mypy is installed - which might be the case for development instances, but is neither the case in docker images, nor in "regular" user's installs (if dev-dependencies have been unselected).

Reply:
My bad. Removed that. Idk how it came to be :D